packages/flutter_tools/test/integration/test_driver.dart

```diff
@@ -23,9 +23,11 @@ const Duration appStartTimeout = Duration(seconds: 120);
 const Duration quitTimeout = Duration(seconds: 10);
 
 class FlutterTestDriver {
-  FlutterTestDriver(this._projectFolder);
+  FlutterTestDriver(this._projectFolder, [String logPrefix]):
```

Comment:
You might want to prefer a named param here instead of a postional one, to make it more readable at call sites.

Reply:
Done!